native_client/c_speech_features/README.md

```diff
@@ -0,0 +1,21 @@
+# C Speech Features
```

Comment:
Let's change the documentation - git submodules are awful at the best of times, and I intended c_speech_features to be a drop-in style library, much like kissfft or libmfcc. What did you want to change in the documentation? (ok to just remove it?)